app/Jobs/RestoreDatabaseJob.php

```
@@ -0,0 +1,193 @@
+            // Send failure notification
+            $team = $this->database->team();
+            if ($team) {
+                $team->notify(new RestoreFailed(
+                    $this->database,
+                    $e->getMessage()
+                ));
+            }
+
+            throw $e;
+        }
+    }
```

PHPMD (2.15.0)

The method handle() has 106 lines of code. Current threshold is set to 100. Avoid really long methods. (undefined)

(ExcessiveMethodLength)